src/intl/messages.js

```diff
@@ -394,8 +394,8 @@ export const messages: { [messageId: string]: MessageType } = {
     message: 'Refresh',
     description: 'Reload data from server',
   },
-  remoteDesktopConnectionSettings: 'Remote Desktop Connection Settings',
-  remoteDesktopViewerConnection: 'Remote Desctop Viewer Connection',
+  remoteDesktopConnectionInstructions: 'Console Instructions',
+  remoteDesktopViewerConnection: 'Remote Desktop Viewer Connection',
```

Comment:
rename `consoleInstructions`

string should match the title, which is now `Console Instructions`